shakenfist/daemons/net.py

```diff
@@ -1,24 +1,20 @@
 import logging
-from logging import handlers as logging_handlers
-import setproctitle
 import time
 
 from shakenfist import config
+from shakenfist.daemons import daemon
 from shakenfist import db
 from shakenfist import net
 from shakenfist import util
 
 
-LOG = logging.getLogger(__file__)
-LOG.setLevel(logging.INFO)
-LOG.addHandler(logging_handlers.SysLogHandler(address='/dev/log'))
+LOG = logging.getLogger(__name__)
 
 
-class monitor(object):
-    def __init__(self):
-        setproctitle.setproctitle('sf net')
-
+class Monitor(daemon.Daemon):
     def _maintain_networks(self):
+        LOG.info("Maintaining networks")
```

Comment:
Why double quotes?

Reply:
Because I'm a numpty. I constantly go back and clean up after my muscle memory forces the mistake, but missed these...
I might need a pre-commit hook.
Standby while I fix them.

Reply:
LOL.